### PR TITLE
Update, fix outdated bugs, and minor compilation errors

### PR DESCRIPTION
**Pull request by nurmerey**
Task in Notion - https://www.notion.so/codingallies/8b321612b6774dd7b6aeef351b86a0ea?v=0e16616abeb94afe8361a535013ba660&p=9197c0eefaf7408f9eb9c47aae75e3d3&pm=s 
- Updated package-lock.json
- Fix Material UI outdated library names - createTheme and justifyContent
- Button nesting bug
- Replaced <text> with <Typography>
- Updated background image text with semi-transparent bg
Screenshot 2023-06-15 at 2 54 11 PM

- Updated cards & added community card
Screenshot 2023-06-15 at 3 51 58 PM

### CODE REVIEW

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+import { Component } from 'react';
+import { Grid, Button, Typography, Paper, ThemeProvider } from '@material-ui/core';
+import { createTheme } from '@material-ui/core/styles'
 import whiteBackground from "../assets/MainPageImage.png"
-import {
-  Link
-} from "react-router-dom";
-import HomeCard from '../components/DataStructureAndAlgorithms';
-import { normalize } from 'path';
+// import {
```
Comment (ceyuen) [inline src/pages/Home.tsx:5]: Is it safe to remove this? What are the intentions of keeping it around?

**Review by ceyuen — COMMENTED** [review]
Did the package.json update too? I only see the package-lock.json updated.

File: src/pages/Home.tsx (hunk 1 above)
Comment (nurmerey) [reply src/pages/Home.tsx:5]: They aren't being used at the moment. We can re-add it when we need to.

Comment (nurmerey) [reply src/pages/Home.tsx:5]: Nah, did not update package.json, I just run 'npm install' and 'npm audit fix', which only updates the package-lock.json.

**Comment by nurmerey** [comment]
Revised Cards looks like this. Let me know what you think, I can revise links and texts in the next PRs.
Screenshot 2023-06-16 at 5 03 02 PM

**Review by ceyuen — APPROVED** [review]

**Pull request merged**